sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/AbstractSynchronousInstrument.java

```diff
@@ -41,7 +42,7 @@ public B bind(Labels labels) {
     }
 
     // Missing entry or no longer mapped, try to add a new entry.
-    binding = boundFactory.apply(instrumentProcessor);
+    binding = boundFactory.apply(instrumentProcessor.getAggregator());
```

Comment:
interesting that the coupling is the reverse of what is in @jmacd's design in the spec. I don't think it's necessarily a problem, but it might point at some tweaks to the SDK spec being necessary.